LEAF_Request_Portal/ajaxIndex.php

```diff
@@ -26,9 +26,16 @@
 $db_phonebook = new DB($config->phonedbHost, $config->phonedbUser, $config->phonedbPass, $config->phonedbName);
 unset($db_config);
 
-function customTemplate($tpl)
+function customTemplate($tpl, $config)

```

Comment:
Since all you're using from the config is portalPath, it would be better to just pass in portalPath, instead of taking the whole config.

Reply:
done